examples/oneview_server_profile_template.yml

```diff
@@ -31,6 +31,8 @@
           serverProfileName: "{{ server_profile_name }}" # Optional - Server Profile to base this SPT on
           serverHardwareTypeName: "{{ server_hardware_type_name }}"
           enclosureGroupName: "{{ enclosure_group_name }}"
+        params:
+          force: "True" # Supprted only for API version >= 600
```

Comment:
Is it a mandatory field? Because as per spec, the default value for force param is True.

Reply:
Its not a mandatory field but this is the first query parameter for this end point and needed some changes in the code to support query parameter.